tests/unit_tests/assets/ssl_certs/localhost.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Probably not a huge deal but our secret scanning and pen testers will flag this.  Ideally we would just generate this dynamically but that will likely come with it's own headaches of not having openssl or the right version.  I wonder if we could add a comment before the private key header saying this is for testing only.

Reply:
Ah, good point. Added comments to both the key and cert!